tests/unit/providers/mariner/test_mariner.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+import logging
+
+import shutil
+
+import pytest
+from pytest_unordered import unordered
+from vunnel import result, workspace
+from vunnel.providers.mariner import Config, Provider, parser
+from vunnel.providers.mariner.parser import MarinerXmlFile
+from vunnel.utils.vulnerability import Vulnerability, FixedIn
+
+
```

Comment:
every provider additionally has a schema-specific test to ensure that the results have the appropriate data shape... can you add that as well?

e.g. https://github.com/anchore/vunnel/blob/bbe25d971eaf537864846494cfe7cce475ca0e00/tests/unit/providers/sles/test_sles.py#L288-L306